diff --git a/build.rs b/build.rs
index 03826ef7..35551b5c 100644
--- a/build.rs
+++ b/build.rs
@@ -1,4 +1,4 @@
 use std::env;
-use std::fs::File;
+use std::fs::{read_to_string, write, File};
 use std::io::Write;
 use std::path::PathBuf;
@@ -19,3 +19,14 @@ fn main() {
     // Embed build-time information, such as the Git hash.
     built::write_built_file().expect("Failed to acquire build-time information");
+    let built_info = read_to_string(out.join("built.rs"))
+        .unwrap()
+        .replace(
+            "GIT_DIRTY: Option<bool> = None",
+            "GIT_DIRTY: Option<bool> = Some(@gitDirty@)",
+        )
+        .replace(
+            "GIT_COMMIT_HASH: Option<&str> = None",
+            "GIT_COMMIT_HASH: Option<&str> = Some(\"@gitCommitHash@\")",
+        );
+    write(out.join("built.rs"), built_info).unwrap();
 }
